scripts/release/publish-docs-artifacts.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Publish material2 docs assets to the material2-docs-content repo
+# material.angular.io will pull from this assets repo to get the latest docs
+
+cd "$(dirname $0)/../../"
+
+docsPath="./dist/docs"
+tempRepoPath="./dist/tempRepo"
+repoUrl="https://github.com/angular/material2-docs-content"
+
+# If the docs directory is not present, generate docs
+if [ ! -d $docsPath ]; then
+  $(npm bin)/gulp docs
+fi
+
+# Get git meta info for commit
+commitSha="$(git rev-parse --short HEAD)"
+commitAuthorName="$(git --no-pager show -s --format='%an' HEAD)"
+commitAuthorEmail="$(git --no-pager show -s --format='%ae' HEAD)"
+commitMessage="$(git log --oneline -n 1)"
+
+# create directory and clone test repo
+rm -rf $tempRepoPath
+mkdir $tempRepoPath
+git clone $repoUrl $tempRepoPath
+
+# Clean out repo directory and copy contents of dist/docs into it
+rm -rf $tempRepoPath/*
+mkdir $tempRepoPath/docs
```

Comment:
Also while we are talking about naming. Can we rename the file to `publish-docs-content.sh`?

> Artifacts sounds like we are building a release of the components